src/main/resources/db/migration/V1_1_6__renamed_note_reminder_setting_only_work.sql

```diff
@@ -0,0 +1,2 @@
+ ALTER TABLE settings
+  RENAME COLUMN empty_note_reminder_only_work TO empty_note_reminder_is_work
```

Comment:
as you are creating one new version with the new feature - one .sql script is sufficient. Only you locally have the issue that the column has the "old" name. But locally you can just delete the database so it is recreated with the "new" name.
So you can delete 1.1.6 and just use the correct name in 1.1.5.

If a user installs the new version, he never knew the old field name.